Userland/DevTools/HackStudio/CMakeLists.txt

```diff
@@ -34,6 +34,7 @@ set(SOURCES
     Git/GitFilesView.cpp
     Git/GitRepo.cpp
     Git/GitWidget.cpp
+    Dialogs/GML/GMLPreviewDialog.cpp
```

Comment:
This needs to be moved up to the other `Dialog/` entries. Also, I don't think the additional `GML/` directory is necessary.